curriculum/challenges/_meta/lab-moon-orbit/meta.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "Build a Moon Orbit using CSS animation",
```

Comment:
```suggestion
  "name": "Build a Moon Orbit",
```

Reply:
@jdwilkin4 

Is it OK to use this naming convention?

Reply:
Yeah, I think this name works 👍